server.js

```diff
@@ -1,38 +1,34 @@
 // TASK 1: Setting up server
 
 
```

Comment:
The variable names PORT and HOSTNAME should be in uppercase since they are constants.